src/app/docs/timepicker/timepicker-docs.component.html

```diff
@@ -7,16 +7,32 @@
     The timepicker module creates a SKY UX-themed input field and picker for users to enter or select times.
   </sky-docs-demo-page-summary>
 
-  <!-- <sky-docs-code-examples
-    [packageDependencies]="{
-      '@skyux/datetime': '*'
-    }"
-  >
-    <sky-docs-code-example
-      sourceCodePath="src/app/public/plugin-resources/code-examples/time-picker/basic/"
+  <sky-docs-demo>
```

Comment:
If I select the 24-hour format and switch back to the 12-hour format, I don't get the options to select AM or PM..

Reply:
Selecting *Reset* doesn't restore the AM/PM options either. I have to refresh the page for that.

Reply:
This is an issue with timepicker; I'll dig in.